### PR TITLE
Align Quarkus version

### PR DESCRIPTION
**Pull request by gastaldi**
Reuse the same Quarkus version for amazon services

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -22,15 +22,13 @@
...
-        <tag>3.2.2.0</tag>
     </scm>
     <properties>
         <compiler-plugin.version>3.10.1</compiler-plugin.version>
         <maven.compiler.release>17</maven.compiler.release>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <quarkus.version>3.3.0</quarkus.version>
-        <quarkus-amazon-services-bom.version>3.2.4.Final</quarkus-amazon-services-bom.version>
+        <quarkus.version>3.2.10.Final</quarkus.version>
```
Comment (otaviojava) [inline pom.xml:31]: Question: why did you decrease the version.

Comment (gastaldi) [reply pom.xml:31]: Because 3.2 (3.2.10.Final) is the LTS (Long Term Support) release of Quarkus

**Review by otaviojava — APPROVED** [review]

**Comment by gastaldi** [comment]
@otaviojava I requested maintainer permissions for you in https://github.com/quarkiverse/quarkiverse-devops/pull/216. I'll merge once the @quarkiverse/quarkiverse-jnosql team approves

**Comment by amoscatelli** [comment]
@gastaldi there was a reason for the version not to be aligned, but it seems upgrading to Quarkus 3.5.x fixed the issue.

Thank you for the contribution

**Comment by amoscatelli** [comment]
@gastaldi I was wrong.
This makes the Quarkus ecosystem CI fail "Build against latest Quarkus version" : 

```
Error: ] Some problems were encountered while processing the POMs:
Error:  Non-resolvable import POM: Could not find artifact io.quarkus.platform:quarkus-amazon-services-bom:pom:999-SNAPSHOT in snapshots-repo (https://oss.sonatype.org/content/repositories/snapshots) @ line 22, column 25
 @ 
Error:  The build could not read 1 project -> [Help 1]
Error:    
Error:    The project io.quarkiverse.jnosql:quarkus-jnosql-keyvalue-dynamodb-parent:3.2.2.2-SNAPSHOT (/home/runner/work/quarkus-jnosql/quarkus-jnosql/current-repo/keyvalue-parent/dynamodb-parent/pom.xml) has 1 error
Error:      Non-resolvable import POM: Could not find artifact io.quarkus.platform:quarkus-amazon-services-bom:pom:999-SNAPSHOT in snapshots-repo (https://oss.sonatype.org/content/repositories/snapshots) @ line 22, column 25 -> [Help 2]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
Error:  [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
```
https://github.com/quarkiverse/quarkus-jnosql/actions/runs/8657256427/job/23739129769

**Comment by amoscatelli** [comment]
I created #93

**Pull request merged**